### PR TITLE
docs: update running_on_testnet

### PR DESCRIPTION
**Pull request by volod-vana**
Documentation updated, I was able to follow it step by step, deploy a DLP & DLP token, register and approve validator and run 2 validators nodes.

image

### CODE REVIEW

File: docs/running_on_testnet.md [hunk 1]
```diff
@@ -127,14 +128,15 @@ npx hardhat deploy --network satori --tags DLPDeploy
...
+7. In `vana-dlp-chatgpt/.env`, add an environment variable `DLP_CONTRACT_ADDRESS=0x...` and `DLP_TOKEN_SATORI_CONTRACT=0x` (replace with the deployed contract addresses).
 
-8. Verify the contracts, so we can interact with them directly in the block explorer:
+8. If you made any changes to smart contracts code, verify the contracts, so you can interact with them directly in the block explorer:
 
 ```bash
 npx hardhat verify --network satori <data_liquidity_pool_address>
 npx hardhat verify --network satori <data_liquidity_pool_token_address> <owner_address>
 ```
+If you didn't make changes, contracts should be verified automatically. Ypu may need to wait a few minutes / refresh the page to see the verification status.
```
Comment (Kahtaf) [inline docs/running_on_testnet.md:139]:
```suggestion
If you didn't make changes, contracts should be verified automatically. You may need to wait a few minutes / refresh the page to see the verification status.
```

**Review by Kahtaf — APPROVED** [review]

**Review by Kahtaf — APPROVED** [review]

File: docs/running_on_testnet.md [hunk 2]
```diff
@@ -88,13 +88,14 @@ Currency: DAT
 ```
 Note you can only use the faucet once per day. Use the testnet faucet available at https://faucet.vana.org to fund your wallets, or ask a DAT holder to send you some test DAT tokens.
 
-Get your wallet address for all three accounts
+Get your wallet private keys for all three accounts and import them into metamask or another wallet of your choice.
```
Comment (tnunamak) [inline docs/running_on_testnet.md:91]: Maybe I'm missing it, but what does importing them into metamask help with?

Comment (volod-vana) [reply docs/running_on_testnet.md:91]: Sending DLPT from owner address to validators is not possible via CLI tool currently

File: docs/running_on_testnet.md [hunk 3]
```diff
@@ -88,13 +88,16 @@ Currency: DAT
...
 ```bash
-jq -r '.address' ~/.vana/wallets/owner/hotkeys/default
-jq -r '.address' ~/.vana/wallets/validator_4000/hotkeys/default
-jq -r '.address' ~/.vana/wallets/validator_4001/hotkeys/default
+jq -r '.privateKey' ~/.vana/wallets/owner/coldkey
+jq -r '.privateKey' ~/.vana/wallets/validator_4000/coldkey
```
Comment (Kahtaf) [inline docs/running_on_testnet.md:94]: Does this work? I thought the coldkey is encrypted, we can't `jq` it directly.

File: docs/running_on_testnet.md (hunk 2 above)
Comment (tnunamak) [reply docs/running_on_testnet.md:91]: You don't need to import a wallet to send tokens to it though?

File: docs/running_on_testnet.md (hunk 3 above)
Comment (tnunamak) [reply docs/running_on_testnet.md:94]: It should only work if you create the coldkey with the `--no_password` option, in which case it gets stored unencrypted.

Comment (volod-vana) [reply docs/running_on_testnet.md:94]: I used `--no_password`, going to update doc to specify that

File: docs/running_on_testnet.md (hunk 2 above)
Comment (volod-vana) [reply docs/running_on_testnet.md:88]: You need to import owner wallet, validators not neccessary, but I think thats good to have them for visibility

**Pull request merged**